src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2TestBase.cs

```diff
@@ -186,7 +186,7 @@ public Http2TestBase()
                 // Emulate transport abort so the _connectionTask completes.
                 Task.Run(() =>
                 {
-                    TestApplicationErrorLogger.LogInformation(0, ex, "ConnectionContext.Abort() was called. Completing _pair.Application.Output.");
```

Comment:
Can we fix it so that everything that logs directly to the `TestApplicationErrorLogger` in a `TestApplicationErrorLoggerLoggedTest`-derived test class logs to `Logger` instead?

Reply:
Not easily. Calling `Logger.Log` will call `TestApplicationErrorLogger.Log` which would then call `Logger.Log` which would...

Ideally, `TestApplicationErrorLogger` would not be accessible to `TestApplicationErrorLoggerLoggedTest` tests and tests would access the messages through a field. I'll push a commit to try that out.